core/src/main/java/com/linecorp/armeria/client/cookie/DefaultCookieJar.java

```diff
@@ -102,6 +108,30 @@ public void set(URI uri, Cookies cookies) {
         }
     }
 
+    @Override
+    public boolean isExpired(Cookie cookie) {
+        return isExpired(cookie, System.currentTimeMillis());
+    }
+
+    @Override
+    public boolean isExpired(Cookie cookie, long currentTimeMillis) {
+        if (!store.containsKey(cookie)) {
+            throw new IllegalArgumentException("cookie not found.");
+        }
+        return isExpired(cookie, store.get(cookie), currentTimeMillis);
```

Comment:
- Could we handle this without `containsKey()`? Just a single `get()` will do.
- Shouldn't we acquire a lock here as well?